project.yml

```diff
@@ -0,0 +1,143 @@
+name: Kiwix
+options:
+  xcodeVersion: 13.0
+  deploymentTarget:
+      iOS: 15.0
```

Comment:
Here we should put a comment explaining what is the rule to apply regarding supported versions… or maybe just a link to the readme file.

Reply:
updated